packages/components/src/components/text-field/text-field.tsx

```diff
@@ -275,9 +278,11 @@ export class TextField {
             disabled={this.disabled}
             readonly={this.readonly}
             autocomplete={this.inputAutocomplete}
-            {...ariaDetailedById}
             {...ariaInvalidAttr}
-            {...(this.helperText ? ariaDescribedByAttr : {})}
+            {...(this.helperText || this.ariaDetailedId
+              ? ariaDescribedByAttr
+              : {})}
+            {...(this.helperText && this.ariaDetailedId ? ariaDetailsAttr : {})}
```

Comment:
This seems to be a duplicate. One of the ternary operates should be enough

Reply:
@amir-ba I've done by analogy: each {} block for one property. they both can be set up to conditions
![image](https://github.com/user-attachments/assets/1c2bbad2-2d0c-4a40-88c0-37b185d7bb76)

Reply:
i mean ternary would be good if it was either ariaDescribedBy or ariaDetails up to conditions. but it can be none, one or both of them

Reply:
I see , then lets keep it as is